solo-chains/runtime/starlight/src/lib.rs

```diff
@@ -2747,6 +2749,85 @@ impl tanssi_initializer::Config for Runtime {
     type SessionHandler = OwnApplySession;
 }
 
+pub struct BabeCurrentBlockRandomnessGetter;
+impl BabeCurrentBlockRandomnessGetter {
+    fn get_block_randomness() -> Option<[u8; 32]> {
+        Babe::author_vrf_randomness()
```

Comment:
comment

Reply:
specifically that we use the previous block randomness

Reply:
Comment added, let me know if it's clear

Reply:
resolved